src/test/java/io/hemin/wien/model/AtomBuiderFactoryTest.java

```diff
@@ -0,0 +1,26 @@
+package io.hemin.wien.model;
+
+import io.hemin.wien.builder.AtomBuilder;
+import io.hemin.wien.builder.PersonBuilder;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+public class AtomBuiderFactoryTest {
+
+    @Test
+    @DisplayName("should build an Atom model using builder factory methods only")
+    public void testEpisodeBuilderFactory() {
+        PersonBuilder personBuilder = Person.builder()
+            .name("name");
+
+        AtomBuilder atomBuilder = Atom.builder()
+            .addAuthorBuilder(personBuilder);
+
+        assertNotNull(atomBuilder.build());
```

Comment:
Would be nice for consistency if you used assertK's fluent assertions API like all existing tests in the project: `assertThat(atomBuilder.build().isNotNull()`. Same in the other tests. IIRC assertK works just fine in Java, too.

Reply:
@mpgirro will leave it to you whether to do this, it's the last note left from my review to tackle, but I'm easy.

Reply:
Actually I had a look last night, I agree we should be consistent, but I'm not so sure about the seamless interop though. Ran into some troubles at least. I'll merge for now, maybe you can have a look if you've got a moment.

Reply:
Yeah low priority anyway, I'd rather do the Spotify ns integration first as soon as I find some spare time — been super busy this last week